React/wbooks/src/Login/layout.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Field, reduxForm } from 'redux-form';
+
+import './style.css';
+
+const required = value => (value ? undefined : 'Requerido');
```

Comment:
all this could be in a utils file, or maybe in an inputValidations.js file in case you need input validations throughout your app. Avoid adding methods/constants in a layout component file.